Quaternion/tests/test_all.py

```diff
@@ -134,6 +134,8 @@ def test_from_eq_vectorized():
 
 def test_from_eq_shapes():
     q = Quat(equatorial=equatorial_23[0, 0])
+    assert np.array(q.ra0).shape == ()
+    assert np.array(q.roll0).shape == ()
```

Comment:
Should we also test q.yaw and q.pitch?  Their shapes should correspond to q.ra0 and q.dec, but the explicit tests probably have value.

Reply:
Eh, I just added them.